scvi/inference/cite.py

```diff
@@ -0,0 +1,215 @@
+import matplotlib.pyplot as plt
+import torch
+from torch.distributions import Poisson, LogNormal, Gamma
+import numpy as np
+
+from scvi.inference import Posterior
+from . import UnsupervisedTrainer
+
+plt.switch_backend('agg')
+
+
+class CitePosterior(Posterior):
+
+    def ll_umi(self, verbose=False):
+        self.mode = 'umi'
+        ll = self.compute_log_likelihood(self.model)
+        if verbose:
+            print("LL UMI: %.4f" % ll)
+        return ll
+
+    def ll_adt(self, verbose=False):
```

Comment:
Better to get rid of these "wrapper" methods that just call other another method, and to remove "verbose" logging functionality. The code will be simpler, for exampke, if users just call `compute_log_likelihood` directly rather than calling `ll` which in turn calls `compute_log_likelihood`.